api/models/member.py

```diff
@@ -73,6 +75,8 @@ class Roles(models.TextChoices):
         max_length=2, choices=Roles.choices, default=Roles.UNAPPROVED
     )
 
+    pronouns = models.TextField(null=True)
+
```

Comment:
I thought we discussed wanting this to be a general info field. If the instance developer wanted it to be pronouns they could, but for other groups they could have it as other information. Was that idea scrapped? Fine by me if sure, just wondering.